components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissions.kt

```diff
@@ -76,4 +109,21 @@ data class SitePermissions(
 
         private val converter = StatusConverter()
     }
+
+    internal operator fun get(permission: Permission): Status {
```

Comment:
This can be useful externally (it can be marked as public).

Reply:
This is similar to what Fenix is doing 
https://github.com/mozilla-mobile/fenix/blob/885e81699e0e945048c9aebe6a455aea4032ceac/app/src/main/java/org/mozilla/fenix/settings/PhoneFeature.kt#L103